im.fluffychat.Fluffychat.metainfo.xml

```diff
@@ -0,0 +1,83 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>im.fluffychat.Fluffychat</id>
+  <name>Fluffychat</name>
+  <summary>FluffyChat - Chat with your friends</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>AGPL-3.0-only</project_license>
+  <recommends>
+    <control>pointing</control>
+    <control>keyboard</control>
+    <control>touch</control>
+  </recommends>
+  <developer_name>fluffychat.im</developer_name>
+  <url type="homepage">https://fluffychat.im</url>
+  <description>
+    <p>Open. Nonprofit. Cute ♥</p>
+    <p>This is an early Linux build of a Flutter based Android app, some features and parts of UI might be broken or missing</p>
+    <p>9 greatest FluffyChat features:</p>
+    <ul>
+      <li>Opensource and open development where everyone can join</li>
+      <li>Nonprofit - FluffyChat is donation funded</li>
+      <li>Cute design and many theme settings including a dark mode</li>
+      <li>Unlimited groups and direct chats</li>
+      <li>FluffyChat is made as simple to use as possible</li>
+      <li>Free to use for everyone without ads</li>
+      <li>FluffyChat can use your addressbook to find your friends or you can use usernames</li>
+      <li>There is no "FluffyChat server" you are forced to use. Use the server you find trustworthy or host your own</li>
+      <li>Compatible with Riot, Fractal, Nekho and all matrix messengers</li>
+    </ul>
+    
+    <p>Join the discussion: #fluffychat:matrix.org</p>
+  </description>
+
+  <categories>
+    <category>Network</category>
+  </categories>
+
+  <screenshots>
```

Comment:
Can you add at least 2 screenshots more?

Reply:
Screenshots will be soon replaced by better chosen ones, I asked the project owner for them :)